otri/filtering/filters/math_filter.py

```diff
@@ -1,42 +1,60 @@
-from ..filter import Filter, Stream, Collection
-from typing import Mapping, Callable
+from ..filter import Filter, Stream, Sequence, Any, Mapping
+from typing import Callable
 
 
 class MathFilter(Filter):
     '''
     Performs a give operation on keys of an item.
     Input:
-        Single stream 
+        Single stream.
     Output:
-        Single stream
+        Single stream.
     '''
 
-    def __init__(self, input_stream: Stream, keys_operations: Mapping[str, Callable]):
+    def __init__(self, input: str, output: str, keys_operations: Mapping[str, Callable]):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5649486)